### PR TITLE
Add Cloud to Street and Microsoft Flood and Cloud dataset

### PR DESCRIPTION
**Pull request by kbgg**
*Description of changes:* This PR adds the Cloud to Street and Microsoft Flood and Cloud dataset

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/c2smsfloods.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Name: Cloud to Street - Microsoft Flood and Clouds Dataset
+Description: This dataset consists of chips of Sentinel-1 and Sentinel-2 satellite data. Each Sentinel-1 chip contains a corresponding label for water and each Sentinel-2 chip contains a corresponding label for water and clouds. Data is stored in folders by a unique event identifier as the folder name. Within each event folder there are subfolders for Sentinel-1 (s1) and Sentinel-2 (s2) data. Each chip is contained in its own sub-folder with the folder name being the source image id, followed by a unique chip identifier consisting of a hyphenated set of 5 numbers. All bands of the satellite data, as well as the labels, and overview images are contained within the chip folder.
+Documentation: https://www.drivendata.org/competitions/81/detect-flood-water/
+Contact: support@cloudtostreet.info
+ManagedBy: "[Radiant Earth Foundation](https://radiant.earth/)"
+UpdateFrequency: Not updated
+Tags:
+  - computer vision
```
Comment (jeffejefe) [inline datasets/c2smsfloods.yaml:8]: @kbgg are these stored as standard GeoTIFFs or COGs? If they are COGs and there is a STAC collection you can add the tags below:
- cog
- stac

File: datasets/c2smsfloods.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - deep learning
+  - machine learning
+  - floods
+License: CC-BY-4.0 https://creativecommons.org/licenses/by/4.0/
+Resources:
+  - Description: Flood and Cloud Training Dataset
+    ARN: arn:aws:s3:::radiant-mlhub/c2smsfloods
+    Region: us-west-2
+    Type: S3 Bucket
```
Comment (jeffejefe) [inline datasets/c2smsfloods.yaml:17]: @kbgg if there is a shareable STAC catalog, you could add the Explore tag with the url as shown in below example.

https://github.com/awslabs/open-data-registry/blob/a44c83bfec775f2d72621480b137c0222a447d6a/datasets/deafrica-landsat.yaml#L38

File: datasets/c2smsfloods.yaml (hunk 1 above)
Comment (kbgg) [reply datasets/c2smsfloods.yaml:8]: Good catch, added the cog tag. We don't have a publicly accessible static STAC catalog for this collection so the stac tag and explore example don't apply.

**Comment by jeffejefe** [comment]
thanks @kbgg!

**Pull request merged**